apps/web/src/server/views/applications/case-documentation/documentation-properties.njk

```diff
@@ -1,113 +1,93 @@
-					{% endset %}
-					{{ govukDetails({
+                <h2 class="govuk-caption-xl">Document properties</h2>
+                <h3 class="govuk-heading-xl">{{ documentationFile.documentName|documentName }}</h3>
+                <ul class="govuk-list">
+                    <li class="govuk-!-margin-top-7">{# documentationFile.description #}</li>
+                    <li class="govuk-!-margin-top-4">Case reference: {{ case.reference }}</li>
+                    <li class="govuk-!-margin-top-4">Document reference number:</li>
+                    <li class="govuk-!-margin-top-4">
+                        {{ documentationFile.type|fileType }},
+                        {{ documentationFile.size|fileSize }}
+                    </li>
+                </ul>
+                <div class="display--flex govuk-!-margin-top-7 govuk-!-margin-bottom-7">
+                    <a class="govuk-button govuk-!-margin-0" href="upload/new/version">
+                        Upload new version
+                    </a>
+                    <a class="govuk-button govuk-button--secondary govuk-!-margin-0 govuk-!-margin-left-4" href="{{ 'document-download'|url({caseId: caseId, documentGuid: documentationFile.guid}) }}">
+                        Download
+                    </a>
+                </div>
+                <nav>
+                    {% set actionLinks %}
+                    <a class="govuk-link display--block" href="/open">Open</a>
+                    <a class="govuk-link display--block" href="/move">Move</a>
+                    <a class="govuk-link display--block" href="{{ 'document'|url({caseId: caseId, folderId: folderId, documentGuid: documentationFile.guid, step: 'delete'}) }}">Delete</a>
+                    {% endset %}
+                    {{ govukDetails({
 						summaryText: "Document actions",
 						html: actionLinks
 					}) }}
-				</nav>
-
-				<div class="govuk-!-margin-top-7">
-
-					{% set properties = [
-						{label: 'Document Name', value: documentationFile.documentName|documentName, link: '#'},
-						{label: 'Description', value: '', link: '#'},
-						{label: 'From', value: '', link: '#'},
-						{label: 'Agent', value: '', link: '#'},
-						{label: 'Case Stage', value: '', link: '#'},
-						{label: 'Webfilter', value: '', link: '#'},
-						{label: 'Date received', value: (documentationFile.receivedDate|datestamp({format: 'dd/MM/yyyy'})), link: '#'},
-						{label: 'Document type', value: '', link: '#'},
-						{label: 'Redaction', value: ('Redacted' if documentationFile.redacted else 'Unredacted') , link: '#'},
-						{label: 'Status', value: documentationFile.status|statusName , link: '#'},
-						{label: 'Exam library reference', value: '', link: '#'}
+                </nav>
+                <div class="govuk-!-margin-top-7">
+                    {% set properties = [
+						{label: 'File name', value: documentationFile.documentName|documentName, link: 'name'},
```

Comment:
i'm assuming `documentationFile.[prop]` will need updating after integration with the API?

Reply:
yes, at the moment only a few values actually exist